api/design/type.go

```diff
@@ -298,3 +298,9 @@ var Versions = ResultType("application/vnd.hub.versions", func() {
 
 	Required("latest", "versions")
 })
+
+var JWTAuth = JWTSecurity("jwt", func() {
+	Description("Secures endpoint by requiring a valid JWT token retrieved via the /auth/login endpoint.")
+	Scope("api:read", "Read-only access")
+	Scope("api:write", "Read and write access")
```

Comment:
Do we need both scopes now ??

Reply:
auth return jwt with both scopes so defining both at once would be okay I guess